.expeditor/verify_private.pipeline.yml

```diff
@@ -448,6 +448,19 @@ steps:
         linux:
           privileged: true
 
+  - label: "iam v2 force-upgrade to v2 with legacy policies"
```

Comment:
```suggestion
  - label: "iam v2 with legacy policies force-upgrade to v2"
```

Reply:
(with the same change below. mostly i'm thinking it makes sense to keep the "state" (v2 with legacy policies) together before the action (upgrade to v2).